pyout.py

```diff
@@ -22,9 +22,6 @@ class Tabular(object):
 
     Parameters
     ----------
-    data : iterable
-        Each element should be a dictionary where keys are the column
-        names and values are the data to write.
     columns : list of str
```

Comment:
By default should just get those from the records.
BUT what would be useful is to allow for a dictionary to map possibly short /lowercased record names into full names, e.g. `{'last': "Last name", 'first': "First name"}`.
This structure could also define the order of them (if `OrderedDict` or a list)

Reply:
noted: #17